cmd/boost/init.go

```diff
@@ -35,6 +46,28 @@ var initCmd = &cli.Command{
 
 		log.Debug("Trying to connect to full node RPC")
 
+		var walletPSD address.Address
+		if wPSD := cctx.String("wallet-publish-storage-deals"); wPSD != "" {
+			var err error
+			walletPSD, err = address.NewFromString(wPSD)
+			if err != nil {
+				return err
```

Comment:
nit: it would be nice to return a descriptive error here so they know which wallet address failed to parse, eg "failed to parse wallet address publish storage deals '%s': %w"